doc/usecases/done/gridlayer.md

```diff
@@ -0,0 +1,21 @@
+# Use Case: Grid Layer
```

Comment:
I don't think that a layer is right, layers are only useful if some elements can be associated to it. In case of grid I would simply add a left toolbar, like for measurement.

Reply:
I wasn't sure how to call this feature and I agree that layer is probably the wrong category.

We should certainly agree on a term for Konva-Layers that are not tied to the backend in some way, since other use cases (e.g. the base layer for measuring distances) will have to implement them as well.
One candidate I thought of would be "pseudo-layer", but that might be a little confusing.
"Overlay" might be another option.

Reply:
That a konvas layer is used is purely an implementation detail, it is not the use case concern.

```suggestion
# Use Case: Grid
```

Reply:
> That a konvas layer is used is purely an implementation detail, it is not the use case's concern.

Sorry, I didn't phrase that correctly. I meant layers that only exist in the frontend.